src/index.ts

```diff
@@ -0,0 +1,37 @@
+import { Module } from '@nuxt/types'
+import Storyblok from 'storyblok-js-client'
+import initOptions, { Options } from './options'
+import setupBuild from './build'
+
+declare module 'vue/types/vue' {
+  interface Vue {
+    $storyapi: Storyblok
+    $storybridge: StoryblokBridge
```

Comment:
I might be mistaken but it looks like there is no neither StoryblokBridge interface declaration nor StoryblokBridge interface import in this file

Reply:
Yeah, you're right. Don't know what happened there. 🤔

Reply:
I'm a little late here, but it's still relevant, I feel like it's actually correct like that. `StoryblokBridge `is under `declare global` in the type declaration file of `storyblok-js-client`, so I think it's available without an explicit import, once `Storyblok `is imported in the file.